mongo/filterstore.go

```diff
@@ -422,19 +422,24 @@ func createUpdateFilterBlueprint(filter *models.Filter, currentTime time.Time) b
 func createUpdateFilterOutput(filter *models.Filter, currentTime time.Time) bson.M {
 
 	var downloads models.Downloads
-
+	state := "created"
```

Comment:
is there a state constant defined somewhere that we can reuse?

Reply:
Yes we should probably add states to the models package like we did for dataset API and make them exported constants, then refer to it like so `models.CreatedState`

Reply:
Done 👍